StPetersburg/IVEf17.xml

```diff
@@ -0,0 +1,301 @@
+            <pubPlace>Hamburg</pubPlace>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is licensed
+            under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+            </availability>
+         </publicationStmt>
+
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0301IV"/>
+                  <idno>IV Papadopulokeramevs 1</idno>
+                  <altIdentifier>
+                     <idno>Turaev cat. III.7</idno>
+                  </altIdentifier>
+                        </msIdentifier>
+
+               <msContents>
+                  <summary/>
+
+                  <msItem xml:id="ms_i1">
+                    <locus from="7r" to="138v"/>
+                    <title type="complete" ref="LIT2701Dawit"/>
+                     <textLang mainLang="gez"/>
+                        </msItem>
+
+                    <msItem xml:id="ms_i1.1">
+                      <locus from="7r" to="112r"/>
+                      <title type="complete" ref="LIT2000Mazmur"/>
+                        <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i1.2">
+                      <locus from="112v" to="122r"/>
+                      <title type="complete" ref="LIT1828Mahale"/>
+                      <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i1.3">
+                       <locus from="123v" to="128v"/>
+                       <title type="complete" ref="LIT2362Songof"/>
+                       <textLang mainLang="gez"/>
+                       </msItem>
+
+
+                    <msItem xml:id="ms_i1.4">
+                      <locus from="129r" to="134v"/>
+                      <title type="complete" ref="LIT2509Weddas"/>
+                      <textLang mainLang="gez"/>
+                      <note>Starts from Monday.</note>
+                    </msItem>
+                    <msItem xml:id="ms_i1.5">
+                      <locus from="129r" to="134v"/>
+                      <title type="complete" ref="LIT2509Weddas"/>
+                      <textLang mainLang="gez"/>
+                      <note>Starts from Monday.</note>
+                    </msItem>
+
+                    <msItem xml:id="ms_i1.6">
+                      <locus from="137v" to="142v"/>
+                      <title type="complete" ref="LIT2807RepCh91"/>
+                      <textLang mainLang="gez"/>
+                      <note> Excerpt from the
+                       Maṣḥafa saʿātāt (salām-praises for the Saviour and
+                       St. Mary for the 1st, 3rd and 9th hour).</note>
+                    </msItem>
+
+
+               </msContents>
+
+               <!--  numbers of folios are not certain -->
```

Comment:
Maybe this uncertainty could be expressed explicitely in a note in extent. As a comment, it will not be seen

Reply:
ok